src/main/java/com/e2e/exercises/exercise1/BaseTest.java

```diff
@@ -1,31 +1,111 @@
 package com.e2e.exercises.exercise1;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.util.Iterator;
+
+import org.json.JSONException;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
+import net.minidev.json.parser.JSONParser;
+
 public class BaseTest {
 
 	@Test
-	public void test1() {
+	public void test1() throws Exception {
+		JSONObject obj = new JSONObject();
+
+		obj.put("device-code", "HZ096YKML");
+		obj.put("device-name", "JBL T160BT Bluetooth Headset");
+		obj.put("device-info", "JBL T160BT Bluetooth Headset - Black, In the Ear");
+		obj.put("stock", "Instock");
+		obj.put("serviceable", new Boolean(true));
+		obj.put("available-quantities", new Integer(100));
+
+		JSONArray colorCodes = new JSONArray();
+		colorCodes.add("red");
+		colorCodes.add("black");
+		colorCodes.add("white");
+		obj.put("color-codes", colorCodes);
+		
+		File theDir = new File("src/main/resources");
+		if (!theDir.exists()){
+		    theDir.mkdirs();
+		}
+
+		try {
+			FileWriter file = new FileWriter("src/main/resources/output.json");
+			file.write(obj.toJSONString());
+			file.close();
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+
+		StringWriter out = new StringWriter();
+		obj.writeJSONString(out);
+
+		String actual = out.toString();
+		System.out.print(actual);
 		
-		/** 
-		 * FIRST TEST COMES HERE:
-		 * 
-		 * Create the below JSON object using any library.
-		 * Print the same in the console and also save the object in a file in src/main/resources.
-		 * Assert the created JSON is as expected.
-		 * */
+	      System.out.println("Pretty Print of JSON:");
+	      //System.out.println(obj.to);
 
+		try {
+			JSONAssert.assertEquals(
```

Comment:
Good but lets add discreet assertions to the JSON instead of string assertion.
For example: Parse the JSON and 
- Check if the array is having all the color codes.
- Validate if the `serviceable` field is a boolean type.